mafContrib/exampleNewStacker.py

```diff
@@ -23,9 +26,7 @@ def __init__(self, expMJDCol='expMJD', raCol='fieldRA', decCol='fieldDec'):
         self.raCol = raCol
         self.decCol = decCol
 
-    def run(self, simData):
-        # Add new columns to simData.
-        simData = self._addStackers(simData)
+    def _run(self, simData):
```

Comment:
Dither stackers can use '_run' (and do not need to add their own column names), rather than 'run' and adding columns explicitly now.